### PR TITLE
Jsc/processing posts

### PR DESCRIPTION
**Pull request by juansc**
I needs to merge cus I hasn't merged for so long.

### CODE REVIEW

File: test/processing/date-processing-test.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+expect = require('chai').expect
+request = require 'request'
+dateProcessor = require '../processing/date-processing'
```
Comment (rachelriv) [inline test/processing/date-processing-test.coffee]: Is this file necessary?

File: test/processing/social-media-api-test.coffee [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+            expect(res.statusCode).to.eql 200
+            done()
+
+    describe 'when making Twitter API calls for a list', ->
+      it 'We get a 200', (done) ->
+        list_id = 222479944
+        requestURL = "#{serverName}#{twitterListURL}#{listName}"
+        request requestURL, (err, res, body) ->
+          expect(res.statusCode).to.eql 200
+          done()
```
Comment (rachelriv) [inline test/processing/social-media-api-test.coffee]: I think we need to name this `*_spec.coffee` so it will be discoverable by the test script.

**Comment by rachelriv** [comment]
@juansc bump

File: test/processing/date-processing-test.coffee (hunk 1 above)
Comment (juansc) [inline test/processing/date-processing-test.coffee]: No we don't. I'll remove it.

File: test/processing/social-media-api-test.coffee (hunk 2 above)
Comment (juansc) [inline test/processing/social-media-api-test.coffee]: Done.

**Comment by juansc** [comment]
I implemented the suggested modifications.

**Comment by rachelriv** [comment]
:+1:

**Comment by rachelriv** [comment]
LGTM
